base/gmp.jl

```diff
@@ -100,6 +94,11 @@ does not abort on huge allocation and throws OutOfMemoryError instead.
 const ALLOC_OVERFLOW_FUNCTION = Ref(false)
 
 function __init__()
+    # Resuscitate GMP.ONE object
+    ONE.alloc, ONE.size, ONE.d = 1, 1, pointer(_ONE)
```

Comment:
I guess this has to be moved up here to ensure code which access `ONE` w/o triggering `libgmp_init` doesn't see a garbage object?

I also was wondering why `ZERO` is not being initialized anymore, but the answer is simply that it was unused and hence removed (well, hopefully no package used it; but arguably they shouldn't)

Reply:
Yes, that's correct on all counts.  I checked JuliaHub code search and found no instances of `GMP.ZERO` so I think we're (hopefully) good to delete it.